### PR TITLE
Display issue task list on project cards

### PR DESCRIPTION
**Pull request by KN4CK3R**
Display the issue task list on project cards.

grafik

### CODE REVIEW

**Comment by silverwind** [comment]
The icon/text spacing in that screenshot looks a bit odd, but not the fault of this PR of course.

**Review by silverwind — APPROVED** [review]
Could maybe share a new subtemplate with issue list, otherwise LGTM.

**Review by denyskon — APPROVED** [review]
We should refactor this issue card to use "flex-text", but it is out of scope for this PR

File: templates/repo/issue/card.tmpl [hunk 1]
```diff
@@ -49,6 +49,13 @@
 		</div>
 		{{end}}
 		{{end}}
+		{{$tasks := .GetTasks}}
+		{{if gt $tasks 0}}
+			<div class="meta gt-my-2">
+				{{svg "octicon-checklist" 16 "gt-mr-2 gt-vm"}}
+				<span class="gt-vm">{{.GetTasksDone}} / {{$tasks}}</span>
+			</div>
+		{{end}}
```
Comment (delvh) [inline templates/repo/issue/card.tmpl:58]: Can we perhaps extract a subtemplate instead of manually adding it each time?

**Review by delvh — APPROVED** [review]
Same as @silverwind

File: templates/repo/issue/card.tmpl (hunk 1 above)
Comment (KN4CK3R) [reply templates/repo/issue/card.tmpl:58]: "each time"?

Comment (delvh) [reply templates/repo/issue/card.tmpl:58]: issue/PR list, milestones, projects, …

Comment (KN4CK3R) [reply templates/repo/issue/card.tmpl:58]: They already share a template (besides projects now). I have little frontend knowledge, someone else must create a subtemplate which works in all places.

Comment (silverwind) [reply templates/repo/issue/card.tmpl:58]: When the HTML content is the same it's trivial to extract a subtemplate, e.g. cut-paste into new file and add `{{template "file"}}` in place.

Comment (KN4CK3R) [reply templates/repo/issue/card.tmpl:58]: Sure, but the classes are totally different.

Comment (silverwind) [reply templates/repo/issue/card.tmpl:58]: Ideally make classes the same, but we can do later I guess then.

**Pull request merged**